express/blocks/sticky-promo-bar/sticky-promo-bar.js

```diff
@@ -16,7 +16,10 @@ import {
 } from '../../scripts/scripts.js';
 
 export default function decorate($block) {
-  const $close = createTag('div', { class: 'close' });
+  const $close = createTag('button', {
+    class: 'close',
+    'aria-label': 'close',
+  });
```

Comment:
use button instead of div, this allows:
- the button to be focusable and draw a focus ring
- add the aria-label, so screen-readers can tell the user what the button does
- button can be _clicked_ by hitting enter.